tests/unit/preprocessor/_regrid_esmpy/test_regrid_esmpy.py

```diff
@@ -240,10 +246,12 @@ def coord(name=None, axis=None):
                 raise CoordinateNotFoundError()
             return self.coords[name]
 
-        def coords(dim_coords=None):
+        def coords(dim_coords=None, dimensions=None):
             """Return coordinates for mock cube."""
             if dim_coords:
                 return []
+            if dimensions == ():
```

Comment:
```suggestion
            if not np.any(dimensions):
```

Reply:
I like `== ()` better because `dimensions` is always a tuple and thus checking for `()` is the clearer way to check if the tuple is empty

Reply:
OK up to you, man :+1: